includes/Core/Authentication/Exception/Google_Proxy_Code_Exception.php

```diff
@@ -22,6 +22,15 @@
  */
 class Google_Proxy_Code_Exception extends Exception {
 
+	/**
+	 * Temporary code for an undelegated proxy token.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var Access_Code
```

Comment:
What does this mean? Is that a type/class anywhere? I can't find a reference to it 🤔

Reply:
It is a string, I will set better description, my bad. Thank you